src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -13,6 +13,7 @@ public static class ScriptConstants
         public const string AzureFunctionsHttpRouteDataKey = "MS_AzureFunctionsHttpRouteData";
         public const string AzureFunctionsHttpRequestAuthorizationLevelKey = "MS_AzureFunctionsAuthorizationLevel";
         public const string AzureFunctionsRequestIdKey = "MS_AzureFunctionsRequestID";
+        public const string AzureFunctionsHttpProxyRoutingDataKey = "MS_AzureFunctionsHttpProxyRoutingData";
```

Comment:
This isn't route data, is it? The similarity with the MS_AzureFunctionsHttpRouteData key above is confusing. This key is for the "original request" right?